pkg/terraform/install.go

```diff
@@ -28,8 +30,25 @@ func (m *Mixin) Install() error {
 		step.Flags = append(step.Flags, builder.NewFlag("input=false"))
 	}
 
-	for _, k := range sortKeys(step.Vars) {
-		step.Flags = append(step.Flags, builder.NewFlag("var", fmt.Sprintf("'%s=%s'", k, step.Vars[k])))
+	if step.CreateVarFile {
```

Comment:
I'm actually wondering if we want to just run this logic all the time (and remove the need for `CreateVarFile`)?  In other words, would there be a use case to explicitly not want this file to be created?

I'm thinking we could always create it on install... parameters/vars that are listed for subsequent actions (upgrade, etc.) would still override values in this file.  Also, I don't see harm in creating this file even if the `tfvars` parameter and output (as illustrated in the added documentation) aren't declared.  What do you think?

Reply:
I'm cautious about it always being on just because of the use case we're not thinking about.  For the most part var precedence for the command line makes it easy to always be on.  Maybe it should just be enabled by default, so it can be turned off?

Reply:
Okay, that sounds like a good compromise to me.  Indeed, I'd vote we enable this by default, with the docs example showing how to specifically disable it, if/when needed.